src/main/webapp/help/scanId.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Enter the Scan ID of the parent scan based on the application and technology you selected earlier. You can retrieve Scan ID from the AppScan on Cloud/AppScan 360° Server.
```

Comment:
We should remove the strange character after 360.

Reply:
Sure, Removed that character.